bolt-contracts/script/holesky/operators/RegisterSymbioticOperator.s.sol

```diff
@@ -14,33 +17,41 @@ contract RegisterSymbioticOperator is Script {
         address symbioticNetwork;
     }
 
-    function run() public {
+    function S01_registerIntoBolt() public {
         uint256 operatorSk = vm.envUint("OPERATOR_SK");
+        string memory rpc = vm.envString("OPERATOR_RPC");
```

Comment:
Why don't we read this from the config file? Or did we get rid of it (`operator.json`)

Reply:
We did